problems/removeStringsEndingInBang.js

```diff
@@ -1,18 +1,24 @@
 /**
-* Removes all strings ending in "!" from an array of strings.
-* 
-* @param {string[]} arr - The input array of strings
-* @returns {string[]} - Returns the strings in arr that don't end with "!"
-*
-* ex: removeStringsEndingInBang(["Hello", "Hi!", "Greetings"])
-* returns: ["Hello", "Greetings"]
-*
-* ex: removeStringsEndingInBang(["!", "", "a"])
-* returns: ["", "a"]
-*/
+ * Removes all strings ending in "!" from an array of strings.
+ *
+ * @param {string[]} arr - The input array of strings
+ * @returns {string[]} - Returns the strings in arr that don't end with "!"
+ *
+ * ex: removeStringsEndingInBang(["Hello", "Hi!", "Greetings"])
+ * returns: ["Hello", "Greetings"]
+ *
+ * ex: removeStringsEndingInBang(["!", "", "a"])
+ * returns: ["", "a"]
+ */
 
-function removeStringsEndingInBang() {
+const removeStringsEndingInBang = (arr) => {
+  let output = [];
+  arr.forEach((el) => {
+    if (!el.includes("!")) {
```

Comment:
We don't care if el includes !, we care if the last char of el is "!" remember how to find the last character of string. And then we can use the strict equality operator.